runner/requirements.live-ai.txt

```diff
@@ -7,7 +6,6 @@ Pillow==10.3.0
 python-multipart==0.0.9
 uvicorn==0.30.0
 huggingface_hub==0.23.2
-xformers==0.0.23
```

Comment:
@victorges @j0sh I had to remove a few dependencies here and added them into `Dockerfile.base-streamdiffusion`. Please check if it does not break liveportrait and streamdiffusion pipeline dependencies.

Reply:
Makes sense to me. I'm out of a dev machine for now but will test later.

Reply:
We also needed to remove xformers from here and install it within our base file for segment-anything-2

Reply:
Thanks 👍

Reply:
FTR needed a small fix for this: https://github.com/livepeer/ai-worker/pull/277